pokemongo_bot/cell_workers/spin_fort.py

```diff
@@ -50,25 +57,16 @@ def work(self):
             player_latitude=f2i(self.bot.position[0]),
             player_longitude=f2i(self.bot.position[1])
         )
-        if 'responses' in response_dict and \
-                'FORT_SEARCH' in response_dict['responses']:
+        if 'responses' in response_dict and 'FORT_SEARCH' in response_dict['responses']:
 
             spin_details = response_dict['responses']['FORT_SEARCH']
             spin_result = spin_details.get('result', -1)
-            if spin_result == 1:
+            if spin_result == SPIN_REQUEST_RESULT_SUCCESS:
```

Comment:
We should make a naming convention for these imo. I don’t like mine so if you got an idea